features/dd-trace-core/consumer-rules.pro

```diff
@@ -1,4 +1,2 @@
 # Do not warn about missing classes required only at compile time by dependencies
--dontwarn org.slf4j.**
 -dontwarn com.google.protobuf.**
```

Comment:
where this one is used?

Reply:
hmmm good question...maybe I do not need it, let me check.